Services/DeviceModels.cs

```diff
@@ -92,5 +94,50 @@ private List<string> GetDeviceModelFiles()
 
             return this.deviceModelFiles;
         }
+
+        public HashSet<string> GetPropNames()
+        {
+            var list = this.GetList().ToList();
+            if (list.Count > 0)
+            {
+                var set = new HashSet<string>();
+                list.ForEach(m =>
+                {
+                    foreach (var item in m.Properties)
```

Comment:
This will only return the initial device properties that are listed in the Device Model, this may not return all of the properties that are actually associated with the device after start-up.

Is that the intended behavior?

Reply:
The properties added after startup will come from the new API located in IOT Hub Manager. The UI will merge both the results.

Reply:
Could you also add a comment explaining the behavior here?

Reply:
done